__tests__/package-hoister.js

```diff
@@ -106,7 +106,7 @@ test('Produces valid destination paths for scoped modules', () => {
     _reference: (({}: any): PackageReference),
   }: any): Manifest);
 
-  const info = new HoistManifest(key, parts, pkg, '', true, false);
+  const info = new HoistManifest(key, parts, pkg, '', true, true, false);
```

Comment:
We really should convert this method to accept and options object instead of boolean arguments. No idea what's happening here :(